Intro_to_Supervised_Learning.md

```diff
@@ -0,0 +1,525 @@
+# Intro to Supervised Learning
+
```

Comment:
lets focus on Regression only as the intro to supervised learning. I would call it" Intro to Regression Analysis: Predicting blood pressure" Why? Because you provided both code and analysis of the output